addons/website/static/src/js/content/menu.js

```diff
@@ -33,6 +33,10 @@ const BaseAnimatedHeader = animations.Animation.extend({
      * @override
      */
     start: function () {
+        // Used to prevent the editor's unbreakable protection from restoring
+        // the menu's auto-hide updates in edit mode.
+        this.el.addEventListener("autoMoreMenu.willAdapt", () => this.options.wysiwyg
+            && this.options.wysiwyg.odooEditor.unbreakableStepUnactive());
```

Comment:
- I see you disable the unbreakable system with `unbreakableStepUnactive`... is it reenabled later automatically?
- Let's use the new syntax for this: `this.options.wysiwyg?.odooEditor.unbreakableStepUnactive()`

Reply:
> I see you disable the unbreakable system with unbreakableStepUnactive... is it reenabled later automatically?

Yes, right after the call of `historyStep()`... details:
The [unbreakableStepUnactive() code](https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L1522-L1527C6) will set both the editor's `_toRollback` and `_checkStepUnbreakable` to `false` ... then `_toRollback` will get the `UNBREAKABLE_ROLLBACK_CODE` value once a [change is detected on unbreakable element](https://github.com/odoo/odoo/blob/master/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L873). Also the `_checkStepUnbreakable` flag is [set to true on every call of historyStep()](https://github.com/odoo/odoo/blob/3528f2c3f9d3b30266f3421672986d202c724a76/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L1117) to unable the "unbreakable" check.

> Let's use the new syntax for this: `this.options.wysiwyg?.odooEditor.unbreakableStepUnactive()`

I think it's only allowed on 16.0+, no?  if so, I will add it on fw-port :+1:

Reply:
> I think it's only allowed on 16.0+, no? if so, I will add it on fw-port 👍

I think we only started to use those after 16.0 release but there should be no reason to not be able to in 15.0 (we support the same browsers there). But you are right to be cautious, we can adapt master only to be extra safe.

Reply:
Yes, Already tested here on [15.0](https://runbot.odoo.com/runbot/build/50949336)  and it seems that runbot is not OK with it :grimacing: (at least for `<=15.0`).